main.nf

```diff
@@ -312,6 +339,36 @@ where_are_my_files = file("$baseDir/assets/where_are_my_files.txt")
 params.run_nuclear_contamination = false
 params.contamination_chrom_name = 'X' // Default to using hs37d5 name
 
+// taxonomic classifer
+params.run_metagenomic_screening  = false
+params.metagenomic_tool = 'malt'
+params.database  = ''
+params.percent_identity = 85
+params.malt_mode = 'BlastN'
+params.malt_alignment_mode = 'SemiGlobal'
+params.malt_top_percent = 1
+params.malt_min_support_mode = 'percent'
+params.malt_min_support_percent = 0.01
+params.malt_min_support_reads = 1
+params.malt_max_queries = 100
+params.malt_memory_mode = 'load'
+params.malt_weighted_lca = false
+
+// maltextract - only including number 
+// parameters if default documented or duplicate of MALT
+params.run_maltextract = false
+params.maltextract_taxon_list = ''
+params.maltextract_ncbifiles = ''
+params.maltextract_filter = "def_anc"
+params.maltextract_toppercent = 0.01
+params.maltextract_destackingoff = false
+params.maltextract_downsamplingoff = false
+params.maltextract_duplicateremovaloff = false
```

Comment:
All of these params should be in `nextflow.config` - I'll do that with the TSV branch unless you want to...

Reply:
As in all default paramaters? I can clean that up this evening.